skopt/learning/tree.py

```diff
@@ -0,0 +1,14 @@
+from sklearn.utils import check_array
+from sklearn.tree import DecisionTreeRegressor as DTRegressor
```

Comment:
`DTRegressor` -> `sklearn_DecisionTreeRegressor`? (to emphasize we are just defining a thin layer over it)